### PR TITLE
Replacing schema with CDDL definition (+ some style consistency)

### LINKED ISSUE #208: Remove "schema" and replace with CDDL
An ask from Henk

### PR DESCRIPTION
**Pull request by henkbirkholz**
fixes #208

Removes the term `schema` and replaces it with `CDDL definition` as well as introduces some style consistency in the proximity of those changes.

### CODE REVIEW

File: draft-ietf-rats-corim.md [hunk 1]
```diff
@@ -1258,26 +1258,25 @@ There are scope-related questions that need to be answered.  ([^tracked-at] http
...
 Non-negative integers are reserved for IANA to assign meaning globally.
 
 ### Data Type Extensions
 
-Data type extensibility has the form `($NAME-type-choice)` where "NAME" is the type name
-and '$' signifies type extensibility.
+Data type extensibility has the form `($NAME-type-choice)` where "NAME" is the type name and '$' signifies type extensibility.
 
-Schema extensions (Map or Data Type) should be documented to facilitate interoperability. CoRIM profiles are best used to document vendor or industry defined extensions.
+New data type extensions MUST be documented to facilitate interoperability.
```
Comment (henkbirkholz) [inline draft-ietf-rats-corim.md:1278]: This okay? Introducing normative language might be out-of-scope of the issue addressed.

Comment (deeglaze) [reply draft-ietf-rats-corim.md:1278]: No, CoRIM profiles can be entirely private and not be meant for use outside of a closed circuit.

Comment (andrew-draper) [inline draft-ietf-rats-corim.md]:
```suggestion
New data type extensions SHOULD be documented to facilitate interoperability.

TODO: Clarify which data types need to be documented in a future change

```

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

**Review by deeglaze — APPROVED** [review]

**Review by andrew-draper — APPROVED** [review]

**Pull request merged**